.github/workflows/_ios-build-test.yml

```diff
@@ -67,27 +67,20 @@ jobs:
           # Install dependencies
           brew install libtool
 
-      - name: Install conda and dependencies
-        run: |
-          # Install conda, setup-miniconda messes with the path that messes with the ruby stuff we do later on
-          curl --retry 3 --retry-all-errors -o "${RUNNER_TEMP}/conda.sh" https://repo.anaconda.com/miniconda/Miniconda3-latest-MacOSX-x86_64.sh
-          chmod +x "${RUNNER_TEMP}/conda.sh"
-          /bin/bash "${RUNNER_TEMP}/conda.sh" -b -p "${RUNNER_TEMP}/anaconda"
-          echo "${RUNNER_TEMP}/anaconda/bin" >> "${GITHUB_PATH}"
-          # shellcheck disable=SC1091
-          source "${RUNNER_TEMP}/anaconda/bin/activate"
-          conda install -y \
-            blas=1.0 \
-            cffi=1.15.1 \
-            cmake=3.22.1 \
-            mkl=2022.1.0 \
-            mkl-include=2022.1.0 \
-            ninja=1.10.2 \
-            numpy=1.23.3 \
-            pyyaml=6.0 \
-            requests=2.28.1 \
-            setuptools=63.4.1 \
-            typing_extensions=4.3.0
+      - name: Setup miniconda for iOS device
+        if: inputs.ios-platform != 'SIMULATOR'
+        uses: pytorch/test-infra/.github/actions/setup-miniconda@main
+        with:
+          python-version: "3.10"
```

Comment:
Is 3.10 the python version that goes with https://repo.anaconda.com/miniconda/Miniconda3-latest-MacOSX-x86_64.sh?  Last time I installed this package, it was 3.9.  But may be that has been changed

Reply:
This PR comes at the right time, we are seeing ios job failures when using python 3.10 coming from this latest package.  So I think we need to stick with 3.9 for this PR.  You can take a look at https://github.com/pytorch/pytorch/pull/91366 for the details

Reply:
Python version was reverted to 3.9.